pdm/cli/actions.py

```diff
@@ -426,9 +429,7 @@ def version_matcher(py_version):
         dict.fromkeys(filter(version_matcher, project.find_interpreters(python)))
     )
     if not found_interpreters:
-        raise NoPythonVersion(
-            f"Python interpreter {python} is not found on the system."
-        )
+        raise NoPythonVersion("Python interpreter is not found on the system.")
     if first or len(found_interpreters) == 1:
         selected_python = found_interpreters[0]
     else:
```

Comment:
I would like to move line 421 to after this line so that it only displays when a decision is actually needed.

Reply:
fixed it here.
2bd80fb